### ISSUE TITLE
--facepalm option (or --use-last-options or similar)

### ISSUE THREAD

**Issue by rosswintle** [wp-cli/ideas#28]
So, here's the thing. I'm lazy (which is why I use wp-cli in the first place, right?!) So I quite often do something like:

`wp core config --prompt`

I work my way through the settings, and I'm normally creating a database as I go and copy-pasting the db name, username, password, etc into the prompts.

If I get to the end of this and I've not set `--skip-check` and I've failed to actually click the "Create database" button, or whatever, the command fails and I need to run the command again, stepping through the options.

What would be GREAT would be if each command I run somehow saved the options passed, and then I could run something like `wp core config --use-last-options` and run the exact same command again with the same parameters I used last time, even if last time I did --prompt.

**Comment by danielbachhuber** [comment]
I've totally done this before — and had a similar idea to introduce the semi-persistent behavior.

It's worth noting that this behavior is most painful with `wp core config` because it includes a large number of arguments. It's less of an issue with other commands.

I don't think introducing a new argument is quite the way to go here for two reasons: 1) decisions, not options, 2) global arguments are incredibly costly because they run the risk of backwards compat issues.

Interested to hear suggested implementations as others have them.

**Comment by trepmal** [comment]
This won't help with remember options entered from `--prompt`, but could be handy otherwise

Say you've run this command:

```
wp core install --url=example.com --title=Example --admin_user=supervisor  --admin_password=strongpassword --admin_email=info@example.com
```

In bash, if you want run it again right away, you can use the "event designator"* to repeat the last command:

```
!!
```

Sometimes I get `core install` and `core config` mixed up, so if I had accidentally typed

```
wp core config --url=example.com --title=Example --admin_user=supervisor  --admin_password=strongpassword --admin_email=info@example.com
```

I could reuse all the flags in a new command like this:

```
wp core install !!:3*
```

(Where `3` is the 3rd "word" in the command last command [`wp` is 0, `core` is 2, etc])

\* scroll to that heading in https://linux.die.net/man/1/bash for more info

**Comment by rosswintle** [comment]
Yeah, aware of how the shell can help. Which is why I prefixed the whole thing with "I'm lazy"!

And I get that global options are big changes.

Perhaps the way to go is to set defaults in my `.yml` file so I just pass the database name and everything else is done (at least in local dev where I can get away with a single user for a lot of things)
